CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# How to Contribute
+
+## Formatting
+
```

Comment:
If we're mentioning buildifier let's wish mention the is ones?

Reply:
I added some doc for `prettier` too.